src/promptflow/dev_requirements.txt

```diff
@@ -1,24 +1,26 @@
-pre-commit
-nox
+azure-identity
+azure-keyvault
+azure-mgmt-storage==21.0.0
```

Comment:
azure-mgmt-storage  is useless. I've proved this in other pipeline

Reply:
good to know this, removed

Reply:
good to know this, removed